src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -7,8 +7,54 @@
 
 namespace System.Reflection.Emit
 {
-    public sealed partial class AssemblyBuilder : Assembly
+    public abstract partial class AssemblyBuilder : Assembly
     {
+        private protected AssemblyBuilder()
```

Comment:
Should this be just `protected`? I do not think `private protected` is going to allow you to inherit from this type outside `CoreLib`.

Reply:
Right